### PR TITLE
[docs] Cloud leftnav and SSL

### PR DESCRIPTION
**Pull request by ddhodge**
Move SSL mode info to Security and add note about 0.0.0.0 IP address

@netlify /latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+menu:
+  latest:
+    identifier: cloud-authentication
+    parent: cloud-secure-clusters
+    weight: 250
+isTocNested: true
+showAsideToc: true
+---
+
+Yugabyte Cloud uses TLS 1.3 for communicating with clusters, and digital certificates to verify the identity of clusters. Using TLS with a digital certificate encrypts the data in transit to ensure a secure connection to your cluster, and prevents man in the middle (MITM) attacks, impersonation attacks, and eavesdropping.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:16]: It's TLS 1.2

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Whether you are connecting to your cluster from a client shell or an application, Yugabyte Cloud clusters require a TLS/SSL connection.
+
+## Download your cluster certificate
+
+Yugabyte Cloud clusters are provisioned with a digital certificate, which is used to verify the identity of the cluster when you connect to it from an application or client.
+
+Connections using YCQL require the use of the certificate. You can connect to a cluster using YSQL without a certificate, although where possible, the certificate should be used.
+
+You can download the certificate and install it on the computer that will be connecting to the cluster as follows:
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:26]: You need to download certificate chain(intermediate CA and root CA) and install ...

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+showAsideToc: true
+---
+
+Yugabyte Cloud uses TLS 1.3 for communicating with clusters, and digital certificates to verify the identity of clusters. Using TLS with a digital certificate encrypts the data in transit to ensure a secure connection to your cluster, and prevents man in the middle (MITM) attacks, impersonation attacks, and eavesdropping.
+
+Whether you are connecting to your cluster from a client shell or an application, Yugabyte Cloud clusters require a TLS/SSL connection.
+
+## Download your cluster certificate
+
+Yugabyte Cloud clusters are provisioned with a digital certificate, which is used to verify the identity of the cluster when you connect to it from an application or client.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:22]: .... with a private key and a certificate signed by Internal Yugabyte Certificate Authority(CA). The certificate is used to ....

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Connections using YCQL require the use of the certificate. You can connect to a cluster using YSQL without a certificate, although where possible, the certificate should be used.
+
+You can download the certificate and install it on the computer that will be connecting to the cluster as follows:
+
+1. In Yugabyte Cloud, select your cluster, and click **Connect**.
+1. Click **YugabyteDB Client Shell** or **Connect to your Application**.
+1. Click **Download CA Cert** to download the cluster root.crt certificate to your computer.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:30]: ... to download the cluster intermediate and root certificates to your computer

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## SSL modes in YSQL
+
+If you are connecting to a cluster using YSQL, the `ysqlsh` shell command and application connection string generated by Yugabyte Cloud use the `verify-full` SSL mode by default to verify the cluster’s identity. `verify-full` requires the use of the cluster certificate.
+
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:35]: cluster certificate = > cluster CA certificate

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Connections using SSL mode `disable` will fail. You can use other SSL modes to connect to clusters as described in the following table.
+
+| sslmode | MITM protection | Notes |
+|---|---|---|
+| allow | No | Effectively works as _require_ (always uses the SSL connection without verification). |
+| prefer | No | Effectively works as _require_ (always uses the SSL connection without verification). |
+| require | No | Uses the SSL connection without verification. You do not need to provide the _sslrootcert_ parameter. |
+| verify-ca | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted certificate authority (CA). Requires the _sslrootcert_ parameter with the path to the cluster certificate. |
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:43]: cluster certificate = > cluster CA certificate

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+| verify-full | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted CA and that the requested server host name matches that in the certificate. Requires the _sslrootcert_ parameter with the path to the cluster certificate. |
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:44]: cluster certificate = > cluster CA certificate

File: docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+If you don't provide an SSL mode, the connection defaults to `prefer`.
+
+For information on SSL modes, refer to [Protection Provided in Different Modes](https://www.postgresql.org/docs/11/libpq-ssl.html#LIBPQ-SSL-PROTECTION) in the PostgreSQL documentation.
+
+## SSL connections in YCQL
+
+Connections to your cluster using YCQL require SSL and the cluster certificate.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-secure-clusters/cloud-authentication.md:52]: cluster certificate = > cluster CA certificate

File: docs/content/latest/yugabyte-cloud/cloud-security/_index.md [hunk 9]
```diff
@@ -1,10 +1,10 @@
...
-headerTitle: Yugabyte Cloud architecture
-linkTitle: Yugabyte Cloud architecture
-description: Security architecture in Yugabyte Cloud.
+title: Security architecture
+headerTitle: Security architecture
+linkTitle: Security architecture
+description: Security architecture of Yugabyte Cloud.
 image: /images/section_icons/index/secure.png
-headcontent: Yugabyte Cloud architecture and security.
+headcontent: Yugabyte Cloud infrastructure security architecture.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-security/_index.md:7]: "... infrastructure security architecture" doesn't sound good. Security architecture generally includes a security aspect of infrastructure as well,

File: docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md [hunk 10]
```diff
@@ -31,13 +31,23 @@ If you have a VPC configured, add one or more IP addresses from the peered VPC t
...
-Yugabyte Cloud clusters require an SSL connection. If you set `sslmode` to `disable`, your connection will fail. Refer to [SSL modes in YSQL](../cloud-connect/connect-client-shell/#ssl-modes-in-ysql).
+Yugabyte Cloud clusters require an SSL connection. If you set `sslmode` to `disable`, your connection will fail. Refer to [SSL modes in YSQL](../cloud-secure-clusters/cloud-authentication/#ssl-modes-in-ysql).
+
+If you are connecting to a cluster using YCQL and see the following error:
+
+```output
+Connection error: ('Unable to connect to any servers', {'44.144.44.4': ConnectionShutdown('Connection to 44.144.44.4 was closed',)})
+```
+
+Ensure you are using the `--ssl` option and the path to the cluster certificate is correct. YCQL connections require the `--ssl` option and the use of the cluster certificate.
```
Comment (proff4eg) [inline docs/content/latest/yugabyte-cloud/cloud-troubleshoot.md:48]: cluster certificate => cluster CA certificate

**Review by aishwarya24 — COMMENTED** [review]
Just a question about ordering.

**Pull request merged**